wpeverest/user-registration
# UR-1865  Feature - Automatically remove registered users who have not activated their accounts.

### All Submissions:






### Changes proposed in this Pull Request:



Closes # .
This feature allows users to automatically delete non activated accounts
### How to test the changes in this Pull Request:

1.GO to global settings and enable Auto Delete Unverified Accounts 
2.Add time period for auto deletion
3.update or reactivate plugin
3.install wp-control and check for ur_delete_non_active_users corn job
4.run corn and verify is users deleted (who's status is not verified and configured time for deletion)



### Types of changes:




### Other information:




### Changelog entry

>  Feature -Automatically remove registered users who have not activated their accounts.
